vulnerabilities/data_dump.py

```diff
@@ -68,3 +68,25 @@ def ubuntu_dump(html):
             vulnerability=vulnerability,
             package=package
         )
+
+
+def archlinux_dump(extract_data):
+    """
+    Save data scraped from archlinux' security tracker.
+    """
+    for data in extract_data:
+        vulnerability = Vulnerability.objects.create(
+            summary=data.get('description', ''),
```

Comment:
You use get() and therefore can create things with no values... it does not make sense IMHO. Also using None is fine. No need to put a default of `''`